tools/pyboard.py

```diff
@@ -482,13 +482,11 @@ def fname_remote(src):
         return src
 
     def fname_cp_dest(src, dest):
-        src = src.rsplit("/", 1)[-1]
+        src = os.path.basename(src)
         if dest is None or dest == "":
             dest = src
-        elif dest == ".":
-            dest = "./" + src
-        elif dest.endswith("/"):
-            dest += src
+        elif dest == "." or dest.endswith("/"):
```

Comment:
Is this intended, seeing you're using os.path functions anywhere improving cross-platform behaviour?

Reply:
Dunno, I was trying to do code-tidyups without changing existing behaviour.